tvm/cell/dict.go

```diff
@@ -503,20 +505,18 @@ func loadLabel(sz uint, loader *Slice, key *Builder) (uint, *Builder, error) {
 		return 0, nil, err
 	}
 
-	bitsLen := uint(math.Ceil(math.Log2(float64(sz + 1))))
-
 	ln, err := loader.LoadUInt(bitsLen)
 	if err != nil {
 		return 0, nil, err
 	}
 
-	var toStore []byte
+	toStore := make([]byte, 1+int(ln)/8)
```

Comment:
It seems less optimal, 2 allocations instead of one, if it is just `toStore = bytes.Repeat` we alloc only once

Reply:
Agreed, I apologize, I'll correct it now.